src/DssVest.t.sol

```diff
@@ -116,7 +115,7 @@ contract DssVestTest is DSTest {
 
         hevm.warp(now + 70 days);
 
-        vest.vest(id, MAX_UINT);
+        vest.vest(id, uint256(-1));
```

Comment:
Is this one still using my function on purpose? It's different than the others.

Reply:
yeah. There are two external vest functions now. This test uses them both.